airflow/dags/payments_views/payments_rides.sql

```diff
@@ -97,15 +97,17 @@ applied_adjustments AS (
 ),
 
 initial_transactions AS (
-    SELECT *
+    SELECT * EXCEPT(route_id),
+        TRIM(route_id) AS route_id
```

Comment:
Same note about trimmed `route_id`. I'd rather we expect to be using cleaned data at this step. I can add a `TRIM` into `stg_cleaned_device_transactions` as a follow-on.

Reply:
If you don't mind me doing it, I'm happy to